### PR TITLE
CMS-304 filer filelist

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- https://divio-ch.atlassian.net/browse/CMS-304

Before: 
screen shot 2015-09-17 at 10 19 23

After:
screen shot 2015-09-17 at 10 19 39

Before:
screen shot 2015-09-17 at 10 29 32

After:
screen shot 2015-09-17 at 10 45 36

Before:
screen shot 2015-09-17 at 10 30 23

After:
screen shot 2015-09-17 at 10 30 15

Before:
screen shot 2015-09-17 at 10 30 49

After:
screen shot 2015-09-17 at 10 30 39

Before:
screen shot 2015-09-17 at 10 31 20

After:
screen shot 2015-09-17 at 10 31 04

Before:
screen shot 2015-09-17 at 10 31 53

After:
screen shot 2015-09-17 at 10 31 41

Before:
screen shot 2015-09-17 at 10 32 27

After:
screen shot 2015-09-17 at 10 32 08

Before:
screen shot 2015-09-17 at 10 33 06

After:
screen shot 2015-09-17 at 10 32 58

Before:
screen shot 2015-09-17 at 10 33 47

After:
screen shot 2015-09-17 at 10 33 35

Before:
screen shot 2015-09-17 at 10 34 13

After:
screen shot 2015-09-17 at 10 34 05

Before:
screen shot 2015-09-17 at 10 35 01

After:
screen shot 2015-09-17 at 10 34 51

### CODE REVIEW

File: djangocms_admin_style/sass/components/_icons.scss [hunk 1]
```diff
@@ -40,6 +50,31 @@ table,
...
+    .movetolink span {
+       @include icon(scissors);
+    }
+    .pastelink {
+        @include icon(clipboard);
+    }
+}
+
+
+
```
Comment (Brankub) [inline djangocms_admin_style/sass/components/_icons.scss]: please remove spaces

**Comment by Brankub** [comment]
LGTM

**Comment by vxsx** [comment]
here https://cloud.githubusercontent.com/assets/2270884/9928701/34ff413c-5d28-11e5-9434-6418d9694d72.png admin is too close to the fields

**Comment by vxsx** [comment]
https://cloud.githubusercontent.com/assets/2270884/9928711/4194dad8-5d28-11e5-9a2f-cc6e20dc77fe.png here i think more padding for table cells would be great

**Comment by vxsx** [comment]
https://cloud.githubusercontent.com/assets/2270884/9928701/34ff413c-5d28-11e5-9434-6418d9694d72.png here also might be good to add more space between field and username
also would be cool to check whats happening with long usernames

**Comment by vxsx** [comment]
but overall this looks amazing

**Comment by lory87** [comment]
@vxsx fixed everything excepting more space between field and username. Could you please explain which space you exactly mean?

screen shot 2015-09-17 at 11 20 08

screen shot 2015-09-17 at 11 22 38

screen shot 2015-09-17 at 11 31 15

**Comment by vxsx** [comment]
LGTM

**Pull request merged**